docs/tex/api/reference.tex

```diff
@@ -0,0 +1,65 @@
+\title{Reference}
+
+{{navbar}}
+
+\subsubsection{Models}
+
+Most edward models are wrappers around tensorflow distrbutions. Therefore, the
```

Comment:
Oh, and similar to the flattening of `Inference` subclasses, I think we can have the random variable subclasses not scaffolded within `RandomVariable`.